alpaca/alpaca_test.go

```diff
@@ -265,6 +266,106 @@ func (s *AlpacaTestSuite) TestAlpaca() {
 		assert.Nil(s.T(), actualLastTrade)
 	}
 
+	// get latest trade
+	{
+		// successful
+		latestTradeJSON := `{
+			"symbol": "AAPL",
+			"trade": {
+				"t": "2021-04-20T12:40:34.484136Z",
+				"x": "J",
+				"p": 134.7,
+				"s": 20,
+				"c": [
+					"@",
+					"T",
+					"I"
+				],
+				"i": 32,
+				"z": "C"
+			}
+		}`
+		expectedLatestTrade := v2.Trade{
+			ID:         32,
+			Exchange:   "J",
+			Price:      134.7,
+			Size:       20,
+			Timestamp:  time.Unix(0, 1618922434484136000),
+			Conditions: []string{"@", "T", "I"},
+			Tape:       "C",
+		}
+		do = func(c *Client, req *http.Request) (*http.Response, error) {
+			return &http.Response{
+				Body: ioutil.NopCloser(strings.NewReader(latestTradeJSON)),
+			}, nil
+		}
+
+		actualLatestTrade, err := GetLatestTrade("AAPL")
+		assert.NotNil(s.T(), actualLatestTrade)
+		assert.Nil(s.T(), err)
```

Comment:
[minor] `NoError` is preferable

Reply:
same for a few more places in this file

Reply:
You are right, I fixed it throughout the file.